index.js

```diff
@@ -14,6 +14,9 @@ function TestExclude (opts) {
     configFound: false
   }, opts)
 
+  if (typeof this.include === 'string') this.include = [this.include]
+  if (typeof this.exclude === 'string') this.exclude = [this.exclude]
```

Comment:
Apologies for the late review - this should probably have just been `this.include = arrify(this.include)` and you can get rid of the extra branches :)